plugins/globalias/globalias.plugin.zsh

```diff
@@ -0,0 +1,11 @@
+globalias() {
+   zle _expand_alias
+   zle expand-word
+   zle self-insert
+}
+zle -N globalias
+bindkey -e " " globalias
+bindkey -v " " globalias
+bindkey -e "^ " magic-space           # control-space to bypass completion
```

Comment:
Separate these bindkeys by a newline and put the comments on top instead of on one side:

```zsh
# space expands all global aliases
bindkey -e " " globalias
bindkey -v " " globalias

# control-space to make a normal space
bindkey -e "^ " magic-space
bindkey -v "^ " magic-space

# use normal space during searches
bindkey -M isearch " " magic-space
```